packages/core/src/database/database.js

```diff
@@ -56,6 +55,26 @@ class Database {
     });
   }
 
+  serialize() {
+    return this.db.serialize();
+  }
+
+  async restore(serializedDb, forced) {
```

Comment:
No need for the `async` keyword if you return a promise straight-away
```suggestion
  restore(serializedDb, forced) {
```